config/common.json

```diff
@@ -1,11 +1,13 @@
 {
     "node": {
         "hostname": "ting.gr",
-        "port": 8080
+        "port": 8080,
+        "password": "apassword"
     },
     "django": {
         "port": 8000,
         "secret_key": "the_big_secret",
+        "password": "apassword",
```

Comment:
This is a shared setting, so we should move it out of these component-specific sections.